### DISCUSSION TITLE
Add inline pg certificates support

### DISCUSSION THREAD

**Discussion by keramblock** [grafana/grafana#38504] [Data Sources]
There is new pg version that allows you to pass inline certificates to pg to connect to db, could you please update that library and add it to ui? Currently there is a multiple problems with pg certs in grafana becuse of:
https://github.com/lib/pq/issues/825 - pg only work for 600 or less permissions
https://github.com/kubernetes/kubernetes/issues/57923 - you could not create such permissions in modern k8s becuse of runAsUser option on helm chart

Here is the same request to vault as an example: 
https://github.com/hashicorp/vault/issues/10925

**Comment by gabor** [comment]
hi @Keramblock , it is possible to just paste in the certificates into the datasource-config page, by setting the `TLS/SSL Method` setting to `Certificate content`.

this should solve your problem.

(if that's not the case, please open a new github issue, and please describe why this is not sufficient. thanks!)
